make-all.sh

```diff
@@ -23,6 +23,6 @@ else
     printf "\n${RED}|*******| SOURCE COMPILATION FAILED |*******|${NC}\n\n"
     exit 1
 fi
-# cp src/TerminalFighter ../
+cp src/TerminalFighter/TerminalFighter ../
```

Comment:
Oh I see, I dunno why when I read this I thought it was copying  a ton of source files. This makes sense.